plugins/hypervisors/ovm3/pom.xml

```diff
@@ -27,11 +27,31 @@
     <relativePath>../../pom.xml</relativePath>
   </parent>
   <dependencies>
-<dependency>
-    <groupId>org.apache.xmlrpc</groupId>
-    <artifactId>xmlrpc-client</artifactId>
-    <version>3.1.3</version>
-</dependency>
+    <dependency>
+	<groupId>xml-apis</groupId>
+	<artifactId>xml-apis</artifactId>
+	<version>1.4.01</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.xmlrpc</groupId>
+      <artifactId>xmlrpc-client</artifactId>
+      <version>3.1.3</version>
+    </dependency>
+    <dependency>
+	<groupId>org.apache.xmlrpc</groupId>
+	<artifactId>xmlrpc-common</artifactId>
+	<version>3.1.3</version>
+    </dependency>
+    <dependency>
+	<groupId>org.apache.commons</groupId>
+	<artifactId>commons-lang3</artifactId>
+	<version>3.4</version>
```

Comment:
I can, will do. We should also upgrade some other components to lang3 as lang is outdated